modules/hosted-che/partials/ref_hosted-che-faq-and-troubleshooting.adoc

```diff
@@ -8,7 +8,7 @@
 
 == FAQs
 
-Eclipse Che hosted by Red Hat is part of the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] offering. 
+IMPORTANT: Eclipse Che hosted by Red Hat is part of the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] offering. 
```

Comment:
Space after "link: "

Reply:
@swatikarmarkar this is adoc syntax for adding a link